src/helpers/Http.php

```diff
@@ -2,8 +2,12 @@
 
 namespace markhuot\craftpest\helpers\http;
 
+use markhuot\craftpest\web\TestableResponse;
 use Pest\Expectation;
 
+/**
+ * @return TestableResponse
```

Comment:
Needs TestCall too for descriptionless tests like this,

```php
get(‘/‘)->assertOk();
```

https://github.com/markhuot/craft-pest/commit/8fe70ccba992c439c97314e90a02581b13a4705e#diff-d31e44f76b3be6674ac0163abb32adda7a6875303b119226d204bce904f89ff4R8

Reply:
This is resolved in `dev-master`, the `TestCase::get()` method can't be called description less, so it only returns a `TestableResponse`,

https://github.com/markhuot/craft-pest/blob/816ed3af6e58780a8d500be574c4ecb0ac36b1b5/src/test/TestCase.php#L90

And the helper (which can be called description less) also returns a `TestCall` since the actual object get proxied.

https://github.com/markhuot/craft-pest/blob/816ed3af6e58780a8d500be574c4ecb0ac36b1b5/src/helpers/Http.php#L8

Please re-open if I'm still not understanding.